README.md

```diff
@@ -1,13 +1,9 @@
 # CoffeeHouse API Wrapper for Python
 
 This is a very simple API Wrapper for the CoffeeHouse API. Using
-This Library only supports the V1IVA2.0 API which is based from
+This Library only supports the V1IVA2.0 Lydia API which is based from
```

Comment:
CoffeeHosue-Python-API-Wrapper is not a single wrapper just for Lydia but rather it is built in such a way where it can be expanded with features that can be added to CoffeeHouse later on without breaking the library after every update.